projects.txt

```diff
@@ -122,7 +122,7 @@ jaraco.windows
 jaraco.xkcd
 jaraco.xonsh
 jaraco.zstd
-keyring [lifted]
+keyring [lifted] (python, keyring, python-keyring, keychain, secrets, dbus)
```

Comment:
This file is not the best location for this metadata. This file primarily aims to define my relationship to the projects. The topics of a project should not be defined per-user, but instead in the project itself. And in fact, Python package metadata already has a "keywords" field, so probably that's where these topics should be derived.

Reply:
Alright, sounds much better. I'll navigate to metadata keywords then, see you there!